hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/utils/FileServiceUtils.java

```diff
@@ -92,4 +112,73 @@ public static void validateAndAddRequiredKeys(
 
         if (listKeys != null && listKeys.hasKeys()) for (Key key : listKeys.keys()) context.requireKey(key);
     }
+
+    /**
+     * This version of the method requires that the file exist in state and not be deleted. If the
+     * file is empty this call will fail with {@link ResponseCodeEnum#INVALID_FILE_ID}. If the
+     * special file id is greater than special file config , this call will fail with {@link
+     * ResponseCodeEnum#INVALID_FILE_ID}. If the file has no keys or empty list, this call will fail
+     * with {@link ResponseCodeEnum#UNAUTHORIZED}. If the file is already deleted, this call will
+     * fail with {@link ResponseCodeEnum#FILE_DELETED}.
+     *
+     * @param handleContext the handle context for the transaction.
+     * @param fileStore the file store to fetch the metadata of specified file id
+     * @param fileId the file id to validate and to fetch the metadata
+     * @param canBeDeleted flag indicating that the file may be deleted, and the call should not
+     *     fail if it is.
+     * @return the file metadata of specific system file id
+     */
+    public static @NonNull File verifySystemFile(
+            @NonNull final HandleContext handleContext,
```

Comment:
Since this is only used to get the config it would be better to just pass the config record to the method

Reply:
sure I will change it

Reply:
done